### ISSUE TITLE
Please allow installation of DevPod Windows UI without requiring local admin privileges

### ISSUE THREAD

**Issue by kdandersen** [loft-sh/devpod#392]
Please allow DevPod to be installed in users App\\Local or similar location, not requiring local admin privileges or elevation on Windows.

**Comment by pascalbreuninger** [comment]
Hi @kdandersen, thank you for opening the issue. We looked into the topic a while ago but tbh it's not very high up on the priority list.
You can try to work around it using `msiexec`, see our [slack channel](https://loft-sh.slack.com/archives/C056ZDZPJ4W/p1684322683328249) for more information on that

**Comment by kdandersen** [comment]
Hmm,

Gave it a try with command: 

`msiexec /i "DevPod_windows_x64_en-US.msi" ALLUSERS="" INSTALLDIR=%LOCALAPPDATA%/Programs/DevPod`

But still required elevation.

Anyhow, thanks for suggesting.

Rgds

**Comment by philippe-granet** [comment]
@pascalbreuninger same problem, many companies impose restrictions on their PC and disable local admin privileges or elevation on Windows for securirty reasons.  
It's my case, and I can't test devpod :(

**Comment by philippe-granet** [comment]
I found a work around.  
Extract the msi: `msiexec /a DevPod_windows_x64_en-US.msi /qb TARGETDIR=c:\devpod`  
Then you find `DevPod.exe` in `C:\devpod\PFiles\DevPod` and can launch it without local admin privileges or elevation

**Comment by kdandersen** [comment]
Hi @philippe-granet 

You workaround works! 👍 

Thanks for sharing

**Comment by gbonnefille** [comment]
If so, what about releasing the .exe alone, near to the .msi?
I'm not so sure where the .exe is before releasing it.
https://github.com/loft-sh/devpod/blob/8417972dcdb78e8c0145e587ee36de34636d6046/.github/workflows/release.yaml#L307

**Comment by gbonnefille** [comment]
Please, note that [Rancher Desktop](https://github.com/rancher-sandbox/rancher-desktop) has an installer proposing a User install. But I fear it is a completely different packager.

**Comment by pascalbreuninger** [comment]
@gbonnefille I fear if we just release the `.exe` file it will be more confusing to figure out which file to download and I'm not sure if the signing will still work without the msi. 
Long-term we should adjust the installer to allow for user installs

**Comment by gbonnefille** [comment]
I certainly misunderstood something, but the `.exe`  I found in the Releases is the cli only.

In order to have a GUI running, I had to disassemble the `.msi` and run the `Path` executable.

**Comment by pascalbreuninger** [comment]
Yes you got that right, I thought you want to release the GUI `.exe` alongside the `.msi`

**Comment by natescherer** [comment]
For anyone who comes across this, I've expanded on @philippe-granet 's solution to create a block of Windows PowerShell that will handle the installation under your user's profile, as well as adding DevPod.exe and devpod-cli.exe to the user PATH, creating a Start Menu shortcut, and setting up DevPod to run at Startup:

```PowerShell
# Install DevPod
New-Item -Path "$env:TEMP\devpod" -ItemType Directory
Invoke-WebRequest -Uri "https://github.com/loft-sh/devpod/releases/latest/download/DevPod_windows_x64_en-US.msi" -OutFile "$env:TEMP\devpod\devpod.msi"
Start-Process msiexec -NoNewWindow -Wait -ArgumentList "/a","$env:TEMP\devpod\devpod.msi","/qb","TARGETDIR=`"$env:TEMP\devpod\extracted`""
Copy-Item -Recurse -Path "$env:TEMP\devpod\extracted\PFiles\DevPod" -Destination "$env:LOCALAPPDATA\Programs\DevPod-ManualInstall" -Force
Remove-Item -Recurse -Path "$env:TEMP\devpod" -Force
# Add 'devpod' and 'devpod-cli' to user PATH
if ($env:PATH -notlike "*$env:LOCALAPPDATA\Programs\DevPod-ManualInstall*") {$env:PATH += ";$env:LOCALAPPDATA\Programs\DevPod-ManualInstall";[Environment]::SetEnvironmentVariable("Path", $env:PATH, [System.EnvironmentVariableTarget]::User)}
# Create Start Menu Shortcut
New-Item -Path "$env:APPDATA\Microsoft\Windows\Start Menu\Programs\DevPod-ManualInstall" -ItemType Directory
$Shell = New-Object -ComObject WScript.Shell
$Shortcut = $Shell.CreateShortcut("$env:APPDATA\Microsoft\Windows\Start Menu\Programs\DevPod-ManualInstall\DevPod.lnk")
$Shortcut.TargetPath = "$env:LOCALAPPDATA\Programs\DevPod-ManualInstall\devpod.exe"
$Shortcut.Save()
# Set to run on Startup
Copy-Item -Recurse -Path "$env:APPDATA\Microsoft\Windows\Start Menu\Programs\DevPod-ManualInstall\DevPod.lnk" -Destination "$env:APPDATA\Microsoft\Windows\Start Menu\Programs\Startup\" -Force
```

**Comment by natescherer** [comment]
Bump to (hopefully) reopen this as it is unresolved.
